bazarr/init.py

```diff
@@ -99,19 +96,20 @@ def is_virtualenv():
                     subprocess.check_output(pip_command, stderr=subprocess.STDOUT)
                 except subprocess.CalledProcessError as e:
                     logging.exception(f'BAZARR requirements.txt installation result: {e.stdout}')
-                    os._exit(1)
+                    os._exit(EXIT_REQUIREMENTS_ERROR)
                 else:
                     logging.info('BAZARR requirements installed.')
 
-                try:
-                    restart_file = io.open(os.path.join(args.config_dir, "bazarr.restart"), "w", encoding='UTF-8')
-                except Exception as e:
-                    logging.error(f'BAZARR Cannot create restart file: {repr(e)}')
-                else:
-                    logging.info('Bazarr is being restarted...')
-                    restart_file.write(str(''))
-                    restart_file.close()
-                    os._exit(0)
+                restart_bazarr()
+                # try:
```

Comment:
This commented block is expected?

Reply:
No.
It was a reminder of what the code was doing before the refactoring.
I will remove it.